packages/react-components/src/components/trace-context-component.tsx

```diff
@@ -265,7 +265,9 @@ export class TraceContextComponent extends React.Component<TraceContextProps, Tr
 
     private renderPlaceHolder() {
         return <div className='no-output-placeholder'>
-            {'Add outputs by clicking on an analysis in the trace explorer view'}
+            {'Trace file loaded successfully.'}
+            <br />
+            {'To see available analyses, view the Trace Explorer tab.'}
```

Comment:
say you're a first time user and read "view the Trace Explorer tab", where would one look for that "tab"? According to [this](https://code.visualstudio.com/docs/getstarted/userinterface), the Trace Explorer would be in the "Activity bar".

But to be really picky... this file is in react-components, we don't know where it will be used, theia, vscode, other, we don't even know that there will be a Trace Explorer tab! That placeholder text should probably come as a prop to the trace-context-component.

Reply:
I definitely agree with your points here. I've made the language a little bit more generic to cover use cases that might exist outside of Theia/VS Code, however it doesn't really address making it easier for first time users to know where to go next. I know there have been talks about potentially getting rid of this intermediate placeholder step altogether and re-working the user flow. Perhaps adding a button underneath the text that activates the Trace Explorer view might help so there is no ambiguity of where the tab/activity bar/trace explorer view is? That being said, it's probably out of the scope of this PR to address those changes so I'll keep the placeholder text as fixed rather than as a prop if that's okay. I can also revert the text entirely and just leave the styling changes if you'd like.